src/vue/forms/CreateIssuanceForm.vue

```diff
@@ -0,0 +1,234 @@
+          :error-message="getFieldErrorMessage('form.email')"
+        />
+      </div>
+    </div>
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          v-model="form.reference"
+          @blur="touchField('form.reference')"
+          id="create-issuance-reference"
+          :error-message="getFieldErrorMessage('form.reference')"
+          :label="'issuance.reference' | globalize"
+        />
+      </div>
+    </div>
+    <div class="app__form-actions">
+      <button
+        v-ripple
+        type="submit"
+        class="issuance-form__submit-btn"
+        :disabled="formMixin.isDisabled"
+      >
+        {{ 'issuance.issue' | globalize }}
+      </button>
+      <button
+        v-ripple
+        type="button"
+        class="issuance-form__cancel-btn"
+        :disabled="formMixin.isDisabled"
+        @click.prevent="closeForm"
+      >
+        {{ 'issuance.cancel' | globalize }}
+      </button>
+    </div>
+  </form>
+</template>
+
+<script>
+import FormMixin from '@/vue/mixins/form.mixin'
+import SelectField from '@/vue/fields/SelectField'
+
+import { Bus } from '@/js/helpers/event-bus'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+
+import { Sdk } from '@/sdk'
+import { base } from '@tokend/js-sdk'
+
+import { required, email, between } from '@validators'
+import { vuexTypes } from '@/vuex'
+import { mapGetters } from 'vuex'
+
+export default {
+  name: 'create-issuance-form',
+  components: {
+    SelectField
+  },
+  mixins: [FormMixin],
+  props: {
+    isShown: { type: Boolean, default: true }
+  },
+  data: _ => ({
+    form: {
+      asset: '',
+      amount: 0.00001,
+      email: '',
+      reference: ''
+    },
+    userOwnedTokens: null
+  }),
+  validations () {
+    return {
+      form: {
+        asset: { required },
+        amount: {
+          required,
+          amount: between(
+            0.000001,
+            this.availableTokensAmount ? this.availableTokensAmount.value : 0
+          )
+        },
+        email: { required, email },
+        reference: { required }
+      }
+    }
+  },
+  computed: {
+    ...mapGetters([
+      vuexTypes.account
+    ]),
+    ownedTokenAssets () {
+      if (this.userOwnedTokens) {
+        return this.userOwnedTokens.map(token =>
+          `${token.details.name} (${token.code})`
+        )
+      }
+    },
+    availableTokensAmount () {
+      if (this.userOwnedTokens && this.form.asset) {
+        const token = this.userOwnedTokens.filter(token =>
+          `${token.details.name} (${token.code})` === this.form.asset
+        )[0]
+        return {
+          value: token.availableForIssuance,
+          currency: token.code
+        }
+      }
+    }
+  },
+  async created () {
+    await this.loadUserOwnedTokens()
+    if (this.ownedTokenAssets.length > 0) {
+      this.form.asset = this.ownedTokenAssets[0]
+    }
+  },
+  methods: {
+    async loadUserOwnedTokens () {
```

Comment:
1. It's confusing that we have both `token` and `asset` terms describing the same entity. Let's use `asset` everywhere

Reply:
2. `loadAssets` will be enough I think. The owned assets are the only type of asset we have in the context